### DISCUSSION TITLE
wkhtmltopdf need web.base.url=http://127.0.0.1:8069

### DISCUSSION THREAD

**Discussion by flachica** [Tecnativa/doodba#521] [Q&A]
As some co-workers have recommended to me, in order for Odoo under Doodba to display the reports generated by wkhtmltopdf, you need to change the web.base.url variable to http://127.0.0.1:8069

Without that change the css cannot be loaded. I even tried putting an extra-host in docker but I think it has cut off access.

Is there a way to avoid having to make this change?

**Comment by pedrobaeza** [comment]
I don't know any way to avoid it, but it's a one-time change, so it's not a big deal, and it's documented if I don't remember bad.

**Reply by pedrobaeza** [reply]
Sorry, I read bad `web.base.url` > `report.url`.

**Comment by hhgabelgaard** [comment]
No, AFAIK you need to set report.url to http://localhost:8069
Which doodba does in preparedb: https://github.com/Tecnativa/doodba/blob/master/bin/preparedb#L13
web.base.url need to be the external accessible url: Otherwise links in mails etc. will be wrong

**Reply by ap-wtioit** [reply]
An alternative solution is to declare whatever url you are using as an alias for odoo:
```yaml
odoo:
  networks:
     default:
        aliases:
          - "yourodoo.example.com" # enables odoo accessing odoo as http://yourodoo.example.com:8069/
```

We switched to using a minimal nginx in the doodba network, as it allows us to use proper longpolling and also wkhtmltopdf to access the same urls as the user when the alias is set on nginx (odoo then can access the nginx as http://yourodoo.example.com/ which then sends the wkhtmltopdf css request to odoo)
